src/global_scheduler/global_scheduler_algorithm.c

```diff
@@ -24,6 +24,7 @@ void handle_local_scheduler_heartbeat(global_scheduler_state *state) {
 
 void handle_new_local_scheduler(global_scheduler_state *state,
                                 db_client_id db_client_id) {
-  local_scheduler local_scheduler = {.id = db_client_id};
+  local_scheduler local_scheduler;
+  local_scheduler.id = db_client_id;
```

Comment:
Good point. I've had trouble with initializations like `local_scheduler = {db_client_id};` in the past when I add new fields (or change the order of fields) because then sometimes the entries in the braces end up corresponding to the wrong fields.. do you know a good way to avoid this?